compiler/sc1.cpp

```diff
@@ -3451,21 +3452,21 @@ methodmap_method_t *parse_method(methodmap_t *map)
   symbol* target = parse_inline_function(map, &type, ident.name, is_native, is_ctor, is_static);
 
   if (!target)
-    return NULL;
+    return nullptr;
 
-  methodmap_method_t *method = (methodmap_method_t *)calloc(1, sizeof(methodmap_method_t));
+  auto method = ke::MakeUnique<methodmap_method_t>(map);
   strcpy(method->name, ident.name);
   method->target = target;
-  method->getter = NULL;
-  method->setter = NULL;
+  method->getter = nullptr;

```

Comment:
Are these needed any longer since you already init to `nullptr` in the `methodmap_method_t` constrcutor

Reply:
Nope, thanks, I'll drop these lines.